docs/BreakingChanges/084 ICommand CanExecuteChanged behaviour change in .NET 4.5.md

```diff
@@ -0,0 +1,25 @@
+## 84: ICommand CanExecuteChanged behaviour change in .NET 4.5
```

Comment:
{
  "Id": "84",
  "Title": "ICommand CanExecuteChanged behaviour change in .NET 4.5",
  "Details": "In the .NET Framework 4.5, a CanExecuteChangedEvent was ignored unless the sender of the event was the same object as the object that raised the event. This bug was fixed in .NET Framework 4.5 servcing updates.",
  "Suggestion": "This bug has been fixed in the .NET Framework 4.5 servicing releases, so it can be avoided by making sure that the .NET Framework is up-to-date or by upgrading to .NET Framework 4.5.1. Alternatively, application code using ICommand can be modified to make sure that the sender when raising a CanExecuteChanged command is the same as the object raising the event.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/751429/wpf-icommand-canexecutechanged-behaviour-change-in-net-4-5",
  "Markdown": null,
  "ApplicableApis": [
    "E:System.Windows.Input.ICommand.CanExecuteChanged"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] E:System.Windows.Input.ICommand.CanExecuteChanged